shared/version/useragent.go

```diff
@@ -9,9 +9,11 @@ import (
 )
 
 // UserAgent contains a string suitable as a user-agent
-var UserAgent = getUserAgent(nil)
+var UserAgent = getUserAgent()
+var userAgentStorageBackends []string
```

Comment:
I'm likely just missing context: Where is that initialized?

Reply:
Ah, `backends` below. Don't see where that belongs to but ok. :)

Reply:
yeah, those vars each have matching public functions to set them, those functions also refresh the actual string.